src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
```suggestion
  constructor(name, health = DEFAULT_HEALTH) {
    this.name = name;
    this.health = health;
```
- You should accept health as well in case different animals have different health, it should be 100 by default
- 100 and 50 are magic values, create constants